docs/02-javascript-api.md

```diff
@@ -25,12 +25,13 @@ async function build() {
   const { output } = await bundle.generate(outputOptions);
 
   for (const chunkOrAsset of output) {
-    if (chunkOrAsset.isAsset) {
+    if (chunkOrAsset.type === 'asset') {
       // For assets, this contains
       // {
-      //   isAsset: true,                 // signifies that this is an asset
+      //   isAsset: true,                 // signifies that this is an asset. Accessing "isAsset" on files in the bundle is deprecated, please use "type === \'asset\'" instead
```

Comment:
From my side, I would be ok with just removing this line, then we do not need to clean it up later. But this is ok as well.